packages/web3-eth-accounts/test/unit/wallet.test.ts

```diff
@@ -190,6 +198,29 @@ describe('Wallet', () => {
 			expect(result).toBeFalsy();
 			expect(wallet).toHaveLength(1);
 		});
+
+		it('should remove account with the index', () => {
+			const account = { address: 'my_address' } as never;
+			wallet.add(account);
+			expect(wallet).toHaveLength(1);
+
+			delete wallet[0];
+
+			// Deleting objects dees not change the length
+			expect(wallet).toHaveLength(1);
+			expect(wallet[0]).toBeUndefined();
+		});
+
+		it('should remove account with array methods', () => {
+			const account = { address: 'my_address' } as never;
+			wallet.add(account);
```

Comment:
Can you add a testcase to remove an account that is added using a privatekey？I think it'll be a more complete test if this [part ](https://github.com/ChainSafe/web3.js/blob/477b566ea87bc0aaf862f3af54a83a8cebd14fad/packages/web3-eth-accounts/src/wallet.ts#L67) of the code gets tested too.

Reply:
Added the extra test case.